tournament.md

```diff
@@ -1,15 +1,66 @@
 Tally the results of a small football competition. Based on an input file
-containing which team played against which and what the outcome was 
+containing which team played against which and what the outcome was
 create a file with a table like this:
 
 ```
 Team                           | MP |  W |  D |  L |  P
 Devastating Donkeys            |  3 |  2 |  1 |  0 |  7
-Allegoric Alaskians            |  3 |  2 |  0 |  1 |  6
+Allegoric Alaskans             |  3 |  2 |  0 |  1 |  6
 Blithering Badgers             |  3 |  1 |  0 |  2 |  3
 Courageous Californians        |  3 |  0 |  1 |  2 |  1
 ```
 
-The outcome should be ordered by points, descending. To keep things simple ties
-are broken by looking at the number of wins and if that doesn't solve anything
-by ordering the team names alphabetically.
+What do those abbreviations mean?
+
+- MP: Matches Played
+- W: Matches Won
+- D: Matches Drawn (Tied)
+- L: Matches Lost
+- P: Points
+
+A win earns a team 3 points. A draw earns 1. A loss earns 0.
+
+The outcome should be ordered by points, descending. Ties are broken by
+first looking at the number of wins then by ordering the team names
```

Comment:
if there's one thing I would say it would be that "looking at" doesn't make clear that it's descending in the number of wins.